miniapp/USERGUIDE.md

```diff
@@ -43,7 +43,8 @@ The SDK is configured via `meta-data` tags in your `AndroidManifest.xml`. The fo
 | Field                        | Datatype| Manifest Key                                           | Optional   | Default  |
 |------------------------------|---------|--------------------------------------------------------|----------- |--------- |
 | Base URL                     | String  | `com.rakuten.tech.mobile.miniapp.BaseUrl`              | ❌         | 🚫        |
-| Is Preview Mode              | Boolean | `com.rakuten.tech.mobile.miniapp.IsPreviewMode`        | ✅         | false     |
+| Is Preview Mode              | Boolean | `com.rakuten.tech.mobile.miniapp.IsPreviewMode`        | ✅         | true      |
```

Comment:
I think `IsPreviewMode` is `false` by default, right?

Reply:
Yeah, sorry about the redundant change, updated.